bin/build

```diff
@@ -8,14 +8,54 @@ echo "LAYERS_DIR: " $LAYERS_DIR
 MW_LAYER="$LAYERS_DIR/middleware"
 echo "MW_LAYER: " $MW_LAYER
 
+# Setup caching layer for middleware node_modules
+NODE_MODULES_DIR="${LAYERS_DIR}/node_modules" # the actual layer where the caching values are saved to
+touch "${NODE_MODULES_DIR}.toml"              # the layer toml file that tells lifecycle to enable cache
```

Comment:
are we guaranteed that `$LAYERS_DIR` exists so this `touch` will always succeed?  Otherwisde, I think the `mkdir -p` should go first, then touch.